draft-bradleylundberg-arkg.md

```diff
@@ -53,7 +53,69 @@ author:
 - fullname: John Bradley
   organization: Yubico
   email: ve7jtb@ve7jtb.com
-    
+
+contributor:
+  fullname: Dain Nilsson
+  organization: Yubico
+
+normative:
+  hkdf: RFC5869
+  RFC2104:
+  RFC3279:
+  BIP32:
+    target: https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki
+    title: BIP 32 Hierarchical Deterministic Wallets
+    author:
+    - name: Pieter Wuille
+    date: 2012
+  SEC1:
+    target: http://www.secg.org/sec1-v2.pdf
+    author:
+    - org: Certicom Research
+    date: 2020
+    title: SEC 1 Elliptic Curve Cryptography
+
+informative:
+  Clermont:
+    target: https://www.cryptoplexity.informatik.tu-darmstadt.de/media/crypt/teaching_1/theses_1/Sebastian_Clermont_Thesis.pdf
+    author:
+    - name: Sebastian A. Clermont
+    - org: Technische Universität Darmstadt
+    date: 2022
+    title: Post Quantum Asynchronous Remote Key Generation. Master's thesis
+  Wilson:
+    author:
+    - name: Spencer MacLaren Wilson
+    - org: University of Waterloo,
+    title: Post-Quantum Account Recovery for Passwordless Authentication. Master's thesis
+    date: 2023
+    target: http://hdl.handle.net/10012/19316
+  WebAuthn-Recovery:
+    author:
+    - name: Emil Lundberg
+    - name: Dain Nilsson
+    title: WebAuthn recovery extension Asynchronous delegated key generation without shared secrets. GitHub
+    date: 2019
+    target: https://github.com/Yubico/webauthn-recovery-extension
+  Frymann2020:
+    author:
+    - name: Nick Frymann
+    - name: Daniel Gardham
+    - name: Franziskus Kiefer
+    - name: Emil Lundberg
+    - name: Mark Manulis
+    - name: Dain Nilsson
+    title: Asynchronous Remote Key Generation - An Analysis of Yubico’s Proposal for W3C WebAuthn. CCS
```

Comment:
Should this not include the rest of the name of the publication? Also colon instead of dash?

```suggestion
    title: "Asynchronous Remote Key Generation: An Analysis of Yubico’s Proposal for W3C WebAuthn. CCS '20: Proceedings of the 2020 ACM SIGSAC Conference on Computer and Communications Security"
```

Reply:
done